apps/feed/index.coffee

```diff
@@ -6,10 +6,18 @@ app = module.exports = express()
 app.set "views", __dirname + "/templates"
 app.set "view engine", "jade"
 
-app.get "/", routes.index
+homePathMiddleware = (req, res, next) -> 
+  return next() unless req.user
+  
+  path = req.user.get('home_path') 
+  return res.redirect 302, path unless path is "/" or !path
```

Comment:
Yeah agreed, was trying to be extra careful to have it still evaluate even if the user doesn't have a `home_path` set. I think too careful to the point of being confusing.